### PR TITLE
InaSAFE Headless related PR

### PR DESCRIPTION
**Pull request by lucernae**
This PR only adds a wrapper function to KeywordIO to read keyword given
its filename (keyword filename)

### CODE REVIEW

**Comment by lucernae** [comment]
@ismailsunni Review this please, thank you.

**Comment by timlinux** [comment]
Can you add a unit test for this too please?

**Comment by ismailsunni** [comment]
Yes, unit test first :)

I usually check how the method works in the unit tests. (what's the input, what's the output.

**Comment by lucernae** [comment]
Hahaha yeah, I should put that first.

**Comment by timlinux** [comment]
@lucernae can you re-touch this - I have updated scrutinizer to be aware of the new repo url

**Comment by timlinux** [comment]
Can we get this passing, testing and merged please @lucernae ?

File: safe/utilities/keyword_io.py [hunk 1]
```diff
@@ -117,6 +120,61 @@ def set_keyword_db_path(self, path):
         """
         self.keyword_db_path = str(path)
 
+    @classmethod
+    def read_keywords_file(cls, filename, keyword=None):
```
Comment (lucernae) [inline safe/utilities/keyword_io.py]: @ismailsunni Could you please review this function?

Comment (ismailsunni) [inline safe/utilities/keyword_io.py]: Please add case where the extension is not `.xml` or `.keywords`. Otherwise, LGTM.

**Pull request merged**